<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TermPrefs</class>
 <widget class="QDialog" name="TermPrefs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Terminal Options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabAppearance">
    <attribute name="title">
     <string>Appearance</string>
    </attribute>
    <widget class="QLabel" name="labelForeground">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Foreground Color</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelBackground">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Background Color</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelWrapText">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Wrap Text</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxWrapMode">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Window</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Page</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxForeground">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxBackground">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnChooseFont">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Font </string>
     </property>
    </widget>
    <widget class="QLabel" name="labelWrapLen">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Page Wrap Length</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxPageWrap">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>20</number>
     </property>
     <property name="maximum">
      <number>80</number>
     </property>
     <property name="value">
      <number>32</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxBuffer">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Buffer Lines</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxBufferLines">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>512</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1024</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2048</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Infinite</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelTabSize">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tab Size</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxTabSize">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>240</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
     <property name="value">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelFont">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>96</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Font Family (size)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxHexMode">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Hexadecimal Input</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxHexDump">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>280</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ASCII-Hex Dump</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabFunction">
    <attribute name="title">
     <string>Function</string>
    </attribute>
    <widget class="QCheckBox" name="cbCReturn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(10)  New Line NL</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbMoveRight">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(4) Move Cursor Right</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbClearToEOL">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(11) Clear to End of Line</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbHomeCursor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(1) Home Cursor</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbClearScreen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(0) Clear Screen</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbMoveDown">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(6) Move Cursor Down</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbNewLine">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(13) Carriage Return CR</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbClearLinesBelow">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(12) Clear Lines Below</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbPositionCursorX">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(14) Position Cursor (x)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbBackSpace">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(8) Backspace</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbTab">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(9) Tab</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbSwapNLCR">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Swap Receive CR/NL</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbBeepSpeaker">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(7) Beep Speaker</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbPositionCursorXY">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(2) Position Cursor X,Y</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbPositionCursorY">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(15) Position Cursor (y)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbMoveUp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(5) Move Cursor Up</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbMoveLeft">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(3) Move Cursor Left</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbEnterIsNL">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter is NL</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbClearScreen16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(16) Clear Screen</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnClearOptions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Options</string>
   </property>
  </widget>
  <action name="actionTermPrefsOk">
   <property name="text">
    <string>termPrefsOk</string>
   </property>
  </action>
  <action name="actionTermPrefsCancel">
   <property name="text">
    <string>termPrefsCancel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
